infra/concourse/Makefile.BUILD

```diff
@@ -14,12 +14,12 @@ BUILD_RUBY_VERSION := 2.6.3
 # Fixing bugs or making trivial changes should be considered a patch release.
 
 DOCKER_TAG_VERSION_TERRAFORM := 2.1.0
-DOCKER_TAG_VERSION_KITCHEN_TERRAFORM := 2.3.0
+DOCKER_TAG_VERSION_KITCHEN_TERRAFORM := 2.4.0
```

Comment:
This should be 3.0.0; changing the WORKDIR breaks existing Makefiles which assume the directory is `/cftl/workdir`.